App/Redux/NotificationRedux.js

```diff
@@ -0,0 +1,29 @@
+/* ------------- Types and Action Creators ------------- */
+
+const ADD_NOTIFICATION = 'ADD_NOTIFICATION'
+const CLEAR_NOTIFICATIONS = 'CLEAR_ALL_NOTIFICATIONS'
+
+export const addNotification = (message) => ({
+  type: ADD_NOTIFICATION,
+  notification: {
+    message,
+    start: Date.now()
+  }
+})
+
+export const clearNotifications = () => ({
+  type: CLEAR_NOTIFICATIONS
+})
```

Comment:
We generally get our type constants generated by our helper function.   I can help with this.